### ISSUE TITLE
Provide package for new debian stable (stretch)

### ISSUE THREAD

**Issue by SleuthKid** [apache/couchdb#781]
As debian stable has now moved to stretch some weeks ago it would be nice to have .deb packages for it in the official couchdb repos at https://apache.bintray.com/couchdb-deb/dists/

Compared to debian jessie, stretch now uses OTP 19 instead of OTP 17.

**Comment by wohali** [comment]
We're waiting on an upstream dependency (Docker ansible images) to be released using stretch so our automated build process can be augmented.

https://github.com/William-Yeh/docker-ansible/pull/31

Please feel free to upvote this issue if you think it will help.

**Comment by wohali** [comment]
This is now unblocked.

**Comment by caralbgm** [comment]
I updated from Jessie to Stretch and CouchDB is broken now. In my case, it tries to use library libicu52 but stretch uses libicu57.

**Comment by wohali** [comment]
We have not released a CouchDB stretch package yet. You will have to wait until that package is released, or compile and install it yourself.

**Comment by rajudev** [comment]
@wohali This is a must have, as there are many people who want it on Debian. Do you want any volunteering efforts for this particular task. I could help here. 
Also any idea, why the Jessie repo for couchdb, is not allowing me to source the package from the repo?

**Comment by wohali** [comment]
Please remember that these are *convenience binaries* and that the only official release of Apache CouchDB is the source code. Anything else is a best-effort basis.

We don't at present build srpms / debian source packages.

If you want to see this move faster, PRs are welcome on these repositories:

https://github.com/apache/couchdb-ci
https://github.com/apache/couchdb-pkg

**Comment by rajudev** [comment]
I see that the initial blocker for getting a Debian stretch image ofdocker-ansible is now resolved already. 
So the delay is now from our side. Let me see if a PR is required for anything here. 
 https://github.com/William-Yeh/docker-ansible/pull/31

**Comment by wohali** [comment]
This is currently blocked by problems building SM1.8.5 on stretch.

**Comment by wohali** [comment]
Everything is good to go here. We are already building test stretch packages as part of our CI process for internal consumption by the development community.

When the next CouchDB release happens (2.1.1, very soon) we will issue packages for stretch.

Thanks for everyone's patience.

**Comment by gabssnake** [comment]
Any news on this issue? Would love to find a stretch package.

Regards,

**Comment by wohali** [comment]
Packages for stretch are released at http://docs.couchdb.org/en/latest/install/unix.html#installation-using-the-apache-couchdb-convenience-binary-packages

**Comment by gabssnake** [comment]
Great! Thank you for your awesome responsiveness.

For future surfers:
```
sudo apt-get install -y curl apt-transport-https
echo "deb https://apache.bintray.com/couchdb-deb stretch main" | sudo tee -a /etc/apt/sources.list
curl -L https://couchdb.apache.org/repo/bintray-pubkey.asc | sudo apt-key add -
sudo apt-get -y update
sudo apt-get install couchdb
```
